README.md

```diff
@@ -97,6 +97,20 @@ We use Code generation to generate our types for our GraphQL queries and mutatio
 - Each query and mutation should have a unique name.
 - Since query analysis for type generation occurs statically we cant place dynamic variables with in query strings we instead have to hard code the variable in the query or pass it in as query variable.
 
+## How to get data for your feature
+If you need more data to be able to test out your feature locally the easiest way to do it is to populate the local db using real data from the staging or production environments.
```

Comment:
i do not think prod and staging environments should be presented here as equals because in my experience, staging data is usually low quality and should not be used for local data. are there scenarios or examples you have in mind when using staging data is better than prod data?